src/factory/HeaderPredicateFactory.ts

```diff
@@ -51,6 +52,25 @@ function arithmeticMeasureLocalIdentifierDeepMatch(
     return predicate(operandHeader, context);
 }
 
+function getArithmeticMeasureOperandIdentifiers(measure: AFM.IMeasure): string[] {
+    if (AFM.isArithmeticMeasureDefinition(measure.definition)) {
+        return measure.definition.arithmeticMeasure.measureIdentifiers;
+    }
+
+    return null;
+}
+
+function getDerivedMeasureMasterMeasureOperandIdentifiers(measure: AFM.IMeasure, afm: IAfm): string[] {
```

Comment:
Use `AFM.IAfm` so you don't need to do `import IAfm = AFM.IAfm;` few lines above

Reply:
Much better